app/posts/register-trainee-teachers/2021-04-08-searching-for-a-school.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Searching for a school
+description: Exploring using an autocomplete to pick schools
+date: 2021-04-08
+related:
+  items:
+  - text: Collecting placement details
+    href: register-trainee-teachers/collecting-placement-details/
+  - text: Publish’s schools autocomplete
+    href: publish-teacher-training-courses/schools-autocomplete/
+  - text: Adding School direct routes
+    href: register-trainee-teachers/school-direct-routes/
+---
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{% from "figure/macro.njk" import appFigure with context %}
+
+Several fields in Register require users to search for a school - placement location, lead school and employing school.
+
+[Our first placements iteration](/register-trainee-teachers/collecting-placement-details/) did not go too deeply into this - we started with an input, but assumed we’d come back and add some kind of lookup.
+
+When testing entering schools as part of the placement data we found that:
+
+* participants accepted the need of using URNs.
+* finding the URN could  be fiddly and slowed participants down if they did not already have it.
+* there was some mistrust of URNs and consistent preference for using the name and postcode to both identify and confirm schools. 
+* participants commented that they would like to see the name of the school to help confirm the URN is the correct one.
+
+>“URNs are unreliable: Because a school has gone from school to trust etc. these numbers might change." P4
```

Comment:
One of these quote marks is wrong. Is it the last one? They should both be smart quotes right?